docs/UsersGuide/source/ConfigNewPlatform.rst

```diff
@@ -375,16 +239,24 @@ Those requirements highlighted in **bold** are included in the NCEPLIBS-external
 
 macOS-specific prerequisites:
 
-* brew install wget
+* brew install gcc@11
 * brew install cmake
+* brew install make
+* brew install wget
 * brew install coreutils
 * brew install pkg-config
-* brew install gnu-sed
+
+..
+   COMMENT: Do we need these last 2? Are there others? (e.g., openssl@3?) Lmod, curl, libtiff
 
 Optional but recommended prerequisites:
 
 * Conda for installing/managing Python packages
 * Bash v4+
 * Rocoto Workflow Management System (1.3.1)
 * **CMake v3.15+**
-* Python packages scipy, matplotlib, pygrib, cartopy, and pillow for graphics
+* Python packages ``scipy``, ``matplotlib``, ``pygrib``, ``cartopy``, and ``pillow`` for graphics
+* Lmod
+
+..
+   COMMENT: Are we supporting any installations that don't use Lmod? Should this come under "required?" Or not because it is not mandatory for running the SRW App (but do we know that?).
```

Comment:
The App is supported (to a lesser extent) on platforms that don't have a scheduler and/or module files, so I would say "Lmod" isn't a requirement.